draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2116,7 +2116,281 @@ uCDN-A                         dCDN-B                         dCDN-C
 	<c>RFCthis</c>
 	</texttable>
 	</section>
-	
+
+	<section title="ObjectList" anchor="object-list"><t>
+		ObjectList is a metadata object describing lists of objects that can be used in the context of CI/T v2 trigger spec, trigger status resources and other contexts as required. The ObjectList object can either embed the lists of objects or point to external URL(s) that hold such lists.  ObjectList allows specification of an object list type, providing instructions on the interpretation of the object list format.</t>  
+
+	<t>
+		ObjectLists MAY be recursive, i.e. including references to secondary manifests, including references to HLS, MPEG-DASH or HSS manifests as well as additional JSON-encoded ObjectLists , etc. The party consuming the object list MUST parse all recursions based on the object list type property. When doing so, the consuming party should also detect potential loops when descendant  ObjectList points back to parent  ObjectList.
+	</t>
+	<t>
+ObjectLists MAY combine regular objects and secondary ObjectLists in the same object.
+		Please note that when embedding non-JSON object lists directly in ObjectList, absolute URLs MUST be provided at all times and the text SHOULD be encoded in accordance with the JSON grammar specification [ECMA404], including explicit newline encoding. When the uCDN accesses ObjectList metadata resources published by the uCDN, the same interface authentication and authorization requirements would apply, as when accessing the interface itself.
```

Comment:
1. We likely need an Informative Reference for ECMA404.
2. I am very unclear about this 2nd line.  First, when is the uCDN accessing its own published metadata (likely the first one of these should be dCDN)?  Second, for the authentication, perhaps it would be best to reference (what is currently in draft 12) Section 12.1?  Outside of that, I'm trying to understand if this idea needs to expand to other types of non-objectlist metadata requests.  What if I am requesting prepopulation of a single metadata object? I don't think we have called out any auth requirements for this--maybe this is needed?

Reply:
This really references the next PR, where it is not only uCDN is using ObjectLists in trigger spec, but dCDN may return Objects Lists in Trigger Status as well as Error.v2 description
I have added text that references section 12.1 for uCDN->dCDN as well as reference to dCDN having to consult source access metadata objects if such are available. 

This is probably incomplete though. 
At a minimum need to provide a reference to MI Source objects and other metadata objects (the upcoming MI SourceExtended). We may need to add an option to provide auth directly in the ObjectList object itself in case uCDN uses origin servers that are not referenced in the Configuration Interface.

Reply:
Noting that the ECMA404 informative reference is now present and looks fine--resolved.

As we discussed earlier today, this is an interesting problem.  MI.Source seems to be an interesting way to address it.  I'm trying to think about a use case where the uCDN might reference an origin server that is not referenced in the CI, however.  It seems like it would need to be for the dCDN to understand how to cache it.

Reply:
Added reference to SourceMetadata from RFC8006